servicetalk-http-netty/src/test/java/io/servicetalk/http/netty/DefaultMultiAddressUrlHttpClientBuilderTest.java

```diff
@@ -44,8 +44,9 @@
     @Test
     void buildWithDefaults() throws Exception {
         StreamingHttpRequester newRequester = HttpClients.forMultiAddressUrl()
-                .ioExecutor(CTX.ioExecutor())
-                .executionStrategy(defaultStrategy(CTX.executor()))
+                .initializer((scheme, address, builder) -> builder
+                        .ioExecutor(CTX.ioExecutor())
+                        .executionStrategy(defaultStrategy(CTX.executor())))
```

Comment:
1. Can we revert these changes and use the same methods on `MultiAddressHttpClientBuilder`?
2. Consider adding a separet test to verify that `MultiAddressHttpClientBuilder` correctly sets a use-provided execution context for `client.executionContext()`.

Reply:
I reverted the changes as requested. I also added a more elaborate test that validates both the external client and the internal one with regards to all the possible execution context settings.